New checkboxes

noahmanger opened the pull request:
## Summary
New checkbox styles based on https://github.com/18F/fec-style/issues/314

Resolves https://github.com/18F/fec-style/issues/320

## Screenshots
![image](https://cloud.githubusercontent.com/assets/1696495/14905438/5cf245b6-0d65-11e6-868b-bb22da33877c.png)


_cc @username_

noahmanger commented:
Ok! Ready for review @jenniferthibault .

jenniferthibault commented:
😞 either I am having issues with my local env or something's not right. I see CodeCov is showing errors, so maybe it's that?

` && npm run less && cd .. rm -rf styleguide && kss-node --config kss-config.json && npm run copy-css'.
npm ERR! This is most likely a problem with the fec-style package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     cd fec-template/ && npm run less && cd .. rm -rf styleguide && kss-node --config kss-config.json && npm run copy-css`

I'm not seeing background colors:
<img width="617" alt="screen shot 2016-04-29 at 7 06 31 pm" src="https://cloud.githubusercontent.com/assets/11636908/14931914/904520fe-0e3d-11e6-92a2-e31d83e2dfc0.png">
<img width="554" alt="screen shot 2016-04-29 at 7 06 36 pm" src="https://cloud.githubusercontent.com/assets/11636908/14931915/90470ffe-0e3d-11e6-9399-346aeba5661a.png">

noahmanger commented:
Aaaand that would be because I didn't commit the changes 🙈 . Try now.

jenniferthibault commented:
That makes more sense now 😄 

Ok! I made some small color adjustments and gave the box areas a little more space on the right edge. Looking good on the stylesheet and filter panel:

<img width="583" alt="screen shot 2016-05-02 at 10 12 13 am" src="https://cloud.githubusercontent.com/assets/11636908/14956660/dc4e0908-104e-11e6-8df9-ea564cf8d635.png">

<img width="539" alt="screen shot 2016-05-02 at 10 12 18 am" src="https://cloud.githubusercontent.com/assets/11636908/14956658/dc4de2f2-104e-11e6-9618-38b7280d7058.png">

<img width="1272" alt="screen shot 2016-05-02 at 10 14 32 am" src="https://cloud.githubusercontent.com/assets/11636908/14956659/dc4dc9d4-104e-11e6-8028-c8ae2cd35000.png">


### But I also found an oops!

Looks like the style is somehow affecting our custom drop-down styles:

<img width="296" alt="screen shot 2016-05-02 at 10 01 12 am" src="https://cloud.githubusercontent.com/assets/11636908/14956684/fd6657a8-104e-11e6-8b87-090d5ef89008.png">

<img width="291" alt="screen shot 2016-05-02 at 10 02 22 am" src="https://cloud.githubusercontent.com/assets/11636908/14956683/fd6588dc-104e-11e6-924f-72ff0a672ae3.png">

noahmanger commented:
Changes look good. But yikes! Good catch. Fixed:

![gif](http://g.recordit.co/yfPZv3NtaS.gif)

Note that the focus state changed as part of this change, I think for the better. Previously, items that were in focus just were highlighted. Now, I removed the highlight, but kept the blue focus ring, which makes it clearer it's focused and not selected.

noahmanger commented:
Hmmm apparently wifi at FEC isn't letting me push to GitHub, so that change isn't in yet. Will sync after I'm on a different network.

jenniferthibault commented:
Awesome, looks good on my end! 

@adborden would you like to take a peek for technical review?

adborden commented on package.json:
```diff
@@ -7,12 +7,11 @@
+    "build": "npm run build-sass && npm run build-js && npm run build-styleguide && npm run copy-images",
     "build-js": "browserify -t hbsfy --debug fec-template/public/init.js > fec-template/public/main.js",
     "build-sass": "node-sass scss/ -o css/",
     "build-styleguide": "cd fec-template/ && npm run less && cd .. rm -rf styleguide && kss-node --config kss-config.json && npm run copy-css",
```
Still a reference to `copy-css` here, so `npm run build-styleguide` fails. Should that just be removed?

noahmanger commented:
Thanks! Fixed @adborden .
